core/UI/itembar.py

```diff
@@ -61,8 +61,15 @@ def draw(self):
         pygame.draw.rect(self.game.screen, (50, 50, 50), itembarrects[6])
         self.game.screen.blit(pygame.transform.scale(Assets.BACKPACK1.value.image, (60, 60)), itembar[7])
 
-        self.helditem = (Settings.Game.WIDTH / 2 - 44, Settings.Game.HEIGHT / 2 - 6)
         for i, item in enumerate(self.game.player.inventory.getslots()):
             self.game.screen.blit(pygame.transform.scale(item.item.image, (60, 60)), itembar[i])
+            if item.item != Items.EMPTY.value:
```

Comment:
Ik bedenk me nu ineens dat het misschien handiger is om te checken of item.quantity groter is dan 1. Zo hoeven we niet te checken of een item empty is en wordt een single item gwn los gerendered zonder nummer. Zo niet, dan approve ik 'm gwn

Reply:
hmm, dat kan handig zijn, maar miss is het wel beter om nog wel een 1 te laten zien bij materialen, dus wat ik ga doen is: als de max_stack > 1 dan :rest: